apps/workspaces/apis/import_settings/triggers.py

```
@@ -67,7 +67,7 @@ def __unset_auto_mapped_flag(self, current_mapping_settings: List[MappingSetting
             if current_setting and current_setting.source_field != source_field:
                 changed_source_fields.append(source_field)
 
-        ExpenseAttribute.objects.filter(workspace_id=self.__workspace_id, attribute_type__in=changed_source_fields).update(auto_mapped=False)
+        ExpenseAttribute.objects.filter(workspace_id=self.__workspace_id, attribute_type__in=changed_source_fields).update(auto_mapped=False, updated_at=datetime.now(timezone.utc))
```

Ruff (0.8.2)

Undefined name `datetime`

(F821)